README.md

```diff
@@ -2,7 +2,7 @@
   <img src="https://cdn.rawgit.com/SwifterSwift/SwifterSwift/master/Assets/logo.svg" title="swifterSwift">
 </p>
 
-[![Build Status](https://api.travis-ci.org/SwifterSwift/SwifterSwift.svg?branch=master)](https://travis-ci.org/SwifterSwift/SwifterSwift)
+![](https://github.com/SwifterSwift/SwifterSwift/workflows/SwifterSwift/badge.svg?branch=master)
```

Comment:
You sure about this? Cos the image doesn't exist (yet, I guess?) And do we want to keep the "Build Status" link?

Reply:
yeah, it just won't appear because it didn't run on master yet, check [here](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/configuring-a-workflow#example-using-the-branch-parameter)

Reply:
What about the link? Can we have one like with Travis? Even if it's just to https://github.com/SwifterSwift/SwifterSwift/actions

Reply:
Sorry, I don't understand what you mean ...

Reply:
Here is an example on how it should be when run on master, https://github.com/LucianoPAlmeida/Alamofire/tree/patch-1

Reply:
Ahh I see now :)

Reply:
Done